docs/src/components/home/GoldSponsors.tsx

```diff
@@ -90,6 +90,11 @@ const BACKLINKs = [
     description: 'Instant delivery Instagram followers.',
     href: 'https://twicsy.com/',
   },
+  {
+    name: 'Quickbooks Tool Hub',
+    description: 'An efficient accounting software package.',
```

Comment:
I've got this description from their website since there's none in Open Collective, should we check if that's ok? @rluzists1

Reply:
@zanivan you work so quick I'm impressed - I'll share the preview with them in the ticket, they did say no logo so not sure how they'll feel about the docs display. Will let you know!

Reply:
sponsor is happy, and looks good on my end 👍

Reply:
It looks like this:

```suggestion
    description: 'QuickBook diagnostic and correction tools',
```

Reply:
@oliviertassinari I got it from their website, actually, in the [product description](https://qbtoolhub.com/#:~:text=QuickBooks%20has%20been,and%20much%20more.). Should we really change it again since the sponsor already agreed with the previous description?

Reply:
They are not QuickBook.

Reply:
Got it

Reply:
ok, hopefully that's fine with the sponsor